joblib/memory.py

```diff
@@ -882,7 +882,13 @@ class Memory(Logger):
             as functions are evaluated.
 
         bytes_limit: int, optional
-            Limit in bytes of the size of the cache.
+            Limit in bytes of the size of the cache. By default, the size of
+            the cache is unlimited.
+
+            .. note::
```

Comment:
The Note is nice but would it be possible to remove easily the `Note:` which is a bit big? See [her](https://1606-654608-gh.circle-artifacts.com/0/doc/generated/joblib.Memory.html).

Reply:
Yeah I saw that, I'll see what I can do ...

Reply:
I gave up and went for text not in CSS mood at all ... it seems something in the alabaster theme